app/src/main/java/com/github/onedirection/EventQueries.java

```diff
@@ -33,15 +39,42 @@ public EventQueries(Database db) {
             CompletableFuture<List<Event>> res2 = cdb.filterWhereGreaterLessEq(EventStorer.KEY_EPOCH_END_TIME, ti, tf, EventStorer.getInstance()); // ti < eventEndTime <= tf
             CompletableFuture<List<Event>> res3 = cdb.filterWhereLess(EventStorer.KEY_EPOCH_START_TIME, ti, EventStorer.getInstance()); // eventStartTime < ti
             CompletableFuture<List<Event>> res4 = cdb.filterWhereGreater(EventStorer.KEY_EPOCH_END_TIME, tf, EventStorer.getInstance()); // eventEndTime > tf
-            return CompletableFuture.allOf(res1, res2, res3, res4).thenApply(ignoredVoid -> {
+            CompletableFuture<List<Event>> recurringEvents = cdb.filterWhereGreaterEq(EventStorer.KEY_RECURRING_PERIOD, new Long(0), EventStorer.getInstance());
```

Comment:
[Problem] Listen to the linter: 
```suggestion
            CompletableFuture<List<Event>> recurringEvents = cdb.filterWhereGreaterEq(EventStorer.KEY_RECURRING_PERIOD, Long.valueOf(0), EventStorer.getInstance());
```
In fact, I think that the following would also work
```suggestion
            CompletableFuture<List<Event>> recurringEvents = cdb.filterWhereGreaterEq(EventStorer.KEY_RECURRING_PERIOD, 0L, EventStorer.getInstance());
```
or possibly even
```suggestion
            CompletableFuture<List<Event>> recurringEvents = cdb.filterWhereGreaterEq(EventStorer.KEY_RECURRING_PERIOD, 0, EventStorer.getInstance());
```
(I don't exactly remember how Java behaves with implicit conversions + boxing).